src/DetailsView/components/start-over-dropdown.tsx

```diff
@@ -88,23 +90,33 @@ export class StartOverDropdown extends React.Component<StartOverProps, StartOver
     }
 
     private getMenuItems(): IContextualMenuItem[] {
-        const { testName, rightPanelConfiguration } = this.props;
-        const items: IContextualMenuItem[] = [
-            {
-                key: 'assessment',
-                name: 'Start over Assessment',
-                onClick: this.onStartOverAllTestsMenu,
-            },
-            {
-                key: 'test',
-                name: `Start over ${testName}`,
-                onClick: this.onStartOverTestMenu,
-            },
-        ];
-
-        return rightPanelConfiguration
-            .GetStartOverContextualMenuItemKeys()
-            .map(key => items.find(item => item.key === key));
+        const {
```

Comment:
This is slightly complicated behavior: basically, we don't show the "test" item in the dropdown depending on which rightPanel we're showing (Overview vs. Testview). However, with the advent of Quick Assess, that differential behavior now also applies to the selection of the switcher (QuickAssess vs Assessment).

As such, both config options/preferences must agree for us to show them. If there's an easier/simpler way to do this without removing the right panel config (which I'd be open to, just not here), I'm open to changing this.

Reply:
Agreed, I don't immedietly see a simpler alterative without some more serious refactoring, so this approach seems fine to me.